### PR TITLE
Responsive nav

### PR DESCRIPTION
**Pull request by pierric62**
Fixed the video bug, and  make navigation responsive.

### CODE REVIEW

**Review by VilmaNilsson — COMMENTED** [review]
Everything looks good, however can you please add a screenshot so I can see what you did :D

**Comment by VilmaNilsson** [comment]
Also, there is now a conflict in your branch you need to solve before I can merge.

File: public/css/navigation.css [hunk 1]
```diff
@@ -202,4 +202,11 @@ header {
    #navGrid {
       width: 100vw;
    }
+   .navLinkBig {
+      margin-top: 10%;
+   }
+   .navLinkSmall {
+      font-size: 18pt !important;
```
Comment (trumtomte) [inline public/css/navigation.css:209]: Try to avoid `!important in the future.

File: views/index.pug [hunk 2]
```diff
@@ -32,7 +32,7 @@ html
             //-    .indexPlanetImg
             //-    p Planet
       .indexWrapperDiv#videoDiv
-         iframe(src="https://www.youtube.com/embed/DZg2zUqIdCI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen)
+         iframe(style="width: 85vw; height: 80vh;" src="https://www.youtube.com/embed/DZg2zUqIdCI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen)
```
Comment (trumtomte) [inline views/index.pug:35]: Since this changes the height/width, we really need to have screen shots in order to see how this affects the design.

**Review by trumtomte — CHANGES_REQUESTED** [review]
Some minor comments.

File: views/index.pug (hunk 2 above)
Comment (pierric62) [reply views/index.pug:35]: for the size font
image

for the video 
image

image

**Pull request merged**